website_snippet_marginless_gallery/static/src/js/website_snippet_marginless_gallery.editor.js

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2015-2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+ * Copyright 2019 Tecnativa - Cristina Martin R.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+odoo.define('website_marginless_gallery.gallery', function (require) {
+    'use strict';
+    var options = require('web_editor.snippets.options');
+
+    options.registry.marginless_gallery = options.Class.extend({
+        auto_remove: function () {
+            return this.$target.data("overlay")
+                .find(".oe_snippet_remove").click();
+        },
+
+        drop_and_build_snippet: function () {
```

Comment:
drop_and_build_snippet doesn't exist anymore in 12.0 therefore options.registry.marginless_gallery does nothing now.

Reply:
Yes, this had the purpose of asking for images height. This code would fail if executed (see line 16, calling an unexisting method). Summarizing: remove this file, it's not needed anymore.